.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
```

Comment:
Oh want to add `sudo: false` at the top here? That allows faster builds on Travis I recently learned.